app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
How about this format?

```
{
  “value_copy_format": “MATERIAL_COLOR_$HUE_$VALUE",
  “value_copy_transform”: “_X”,
}
```

All possible values for transform:
`(_|-|t)?(x|X|Xx)?`

Examples:
`“_x”, “-x”, “_X”, “-X”, “_Xx”, “-Xx”, “tx”, “tX"` etc

Whereas, 

`_` replaces space with underscore within variables
`-` replaces space with hiphen within variables
`t` replaces removes space within variables
`x/X/xX` transforms the variables to lower, upper and capitalize respectively.

if `-` or `_` are not specified it doesn’t affect the default variable value (ex: Deep Blue).

Examples:
 - `QUANTUM_COLOR_$HUE_$VALUE` with `_X`: `QUANTUM_COLOR_DEEP_BLUE_500`
 - `$HUE $VALUE` with `Xx`: `Deep Blue 500`
 - `$color$HUE$VALUE` with `tXx`: `$colorDeepBlue500`